### PR TITLE
Facts about Dolphin, Mink and Beaver added, for #298

### PR DESCRIPTION
**Pull request by shreyateeza**
Adds a part of issue #298

### CODE REVIEW

File: animalfacts.py [hunk 1]
```diff
@@ -1016,6 +1017,7 @@ def animalfactsbot(reddit):
     )
 
 DOLPHIN_FACTS = (
+	'Known for their Playful behavior, dolphins are highly Intelligent. They are as Smart as Apes, and the evolution of their larger brains is surprisingly similar to humans.',
```
Comment (mpentler) [inline animalfacts.py]: Can you fix the capital I please on intelligent? And the S on smart?

File: animalfacts.py [hunk 2]
```diff
@@ -1825,6 +1827,7 @@ def animalfactsbot(reddit):
     )
 
 MINK_FACTS = (
+	'Mink is a mammal that belongs to the family of weasels.',
```
Comment (mpentler) [inline animalfacts.py]: Should really be "The mink" if you could?

**Comment by mpentler** [comment]
Hellooo!?

**Comment by shreyateeza** [comment]
@mpentler I will fix it in a day.Thanks for pointing them out :D

File: animalfacts.py (hunk 1 above)
Comment (shreyateeza) [reply animalfacts.py]: Corrected!

File: animalfacts.py (hunk 2 above)
Comment (shreyateeza) [reply animalfacts.py]: Corrected!

**Comment by shreyateeza** [comment]
@mpentler done with the changes!

**Comment by mpentler** [comment]
I fixed a few more small issues but happy to take this now :)

**Pull request merged**